Source/ACE.Database/WorldDatabase.cs

```diff
@@ -150,7 +150,11 @@ public List<AceObject> GetObjectsByLandblock(ushort landblock)
                 o.TextureOverrides = GetAceObjectTextureMaps(o.AceObjectId);
                 o.AnimationOverrides = GetAceObjectAnimations(o.AceObjectId);
                 o.PaletteOverrides = GetAceObjectPalettes(o.AceObjectId);
-                o.AceObjectPropertiesPositions = GetAceObjectPositions(o.AceObjectId).ToDictionary(x => (PositionType)x.DbPositionType, x => new Position(x));
+                // o.AceObjectPropertiesPositions = GetAceObjectPositions(o.AceObjectId).ToDictionary(x => (PositionType)x.DbPositionType, x => new Position(x));
+                // The following three lines is code drafted by Og II for the purpose of linking a weenie's position data to an instance's position data.
+                Dictionary<PositionType, Position> instancePositions = GetAceObjectPositions(o.AceObjectId).ToDictionary(x => (PositionType)x.DbPositionType, x => new Position(x));
+                Dictionary<PositionType, Position> weeniePositions = GetAceObjectPositions(o.WeenieClassId).ToDictionary(x => (PositionType)x.DbPositionType, x => new Position(x));
+                o.AceObjectPropertiesPositions = instancePositions.Union(instancePositions).Union(weeniePositions).ToDictionary(k => k.Key, v => v.Value); // not deduped ?
```

Comment:
I thought about this some more overnight.

I would be more a fan of having a "WeeniePositions" property of ace object (also a Dictionary just like the current "Positions" property).  The Portal class would then look at WeeniePositions[PositionType.Destination] to figure out where to send someone.

I'm really not a fan of making our data fungible and function in different ways based on the class it gets loaded into.